packages/shared/src/server/repositories/observations.ts

```diff
@@ -38,6 +38,30 @@ import {
   TRACE_TO_OBSERVATIONS_INTERVAL,
 } from "./constants";
 
+export const checkObservationExists = async (
```

Comment:
Can we add a start_time filters by using the assumption that evals happen within a couple hours of receiving a trace/observation?